Libplanet.Types/Consensus/IVoteMetadata.cs

```diff
@@ -39,7 +39,7 @@ public interface IVoteMetadata
         /// <summary>
         /// The voting power of the validator that voted.
         /// </summary>
-        BigInteger ValidatorPower { get; }
+        BigInteger? ValidatorPower { get; }
```

Comment:
Don't need any changelog change about this?

Reply:
Amended in https://github.com/planetarium/libplanet/pull/3813/commits/9e25244f1bd5eeef21a5e6479ef346540e5b9223